addons/pos_hr/static/src/app/login_screen/login_screen.js

```diff
@@ -24,6 +25,17 @@ export class LoginScreen extends Component {
             },
             exclusive: true, // takes exclusive control on the barcode reader
         });
+        this.timeFormat = localization.timeFormat;
+        let dateFormat = localization.dateFormat;
+        const splitDateFormat = dateFormat.split(/[^a-zA-Z]+/);
+        for (const format of splitDateFormat) {
+            if (format === "MM") {
+                dateFormat = dateFormat.replace("MM", "LLLL");
+            } else if (format === "yy") {
+                dateFormat = dateFormat.replace("yy", "yyyy");
+            }
+        }
+        this.dateFormat = dateFormat.replace(/[^a-zA-Z]+/g, ", ");
```

Comment:
the reason for this change is to have the date before the month or the other way around, depending on the localization, right?

In that case, do you think we could simplify this ? Something like
```js
this.dateFormat = localization.dateFormat
    .replace(/MM/g, "LLLL")  
    .replace(/\/yy$/, "/yyyy")
    .replace(/[^a-zA-Z]+/g, ", ");
```

Reply:
That would indeed be better I'll try that